cl/simple_pages/tests.py

```diff
@@ -23,3 +30,77 @@ def test_robots_page(self):
         r = self.client.get('/robots.txt')
         self.assertTrue(r.status_code, 200)
 
+
+@override_settings(
+    MEDIA_ROOT = os.path.join(settings.INSTALL_ROOT, 'cl/assets/media/test/')
+)
+class StaticFilesTest(TestCase):
+
+    fixtures = ['court_data.json']
+
+    good_mp3_path = 'mp3/2014/06/09/ander_v._leo.mp3'
+    good_txt_path = 'txt/2015/12/28/opinion_text.txt'
+    good_pdf_path = 'pdf/2013/06/12/' + \
+     'in_re_motion_for_consent_to_disclosure_of_court_records.pdf'
+
+    def setUp(self):
+        self.court = Court.objects.get(pk='test')
+        self.docket = Docket(case_name=u'Docket', court=self.court)
+        self.docket.save()
+
+        self.audio = Audio(
+            local_path_original_file=self.good_mp3_path,
+            local_path_mp3=self.good_mp3_path,
+            docket=self.docket,
+            blocked=False,
+            case_name_full='Ander v. Leo',
+            date_created=datetime.date(2014, 6, 9)
+        )
+        self.audio.save(index=False)
+
+        self.opinioncluster = OpinionCluster(
+            case_name=u'Hotline Bling',
```

Comment:
Nice. Well played. Whatever you do, don't listen to the various testing files that are already in the code base.